plutus-wallet-api/ledger/Ledger/Index.hs

```diff
@@ -91,7 +91,9 @@ data ValidationError =
     | TxOutRefNotFound TxOutRef
     -- ^ The transaction output consumed by a transaction input could not be found (either because it was already spent, or because
     -- there was no transaction with the given hash on the blockchain).
-    | InvalidScriptHash DataScript
+    | InvalidScriptHash ValidatorScript
+    -- ^ For pay-to-script outputs: the validator script provided in the transaction input does not match the hash specified in the transaction output.
+    | InvalidDataHash DataScript DataScriptHash
     -- ^ For pay-to-script outputs: the validator script provided in the transaction input does not match the hash specified in the transaction output.
```

Comment:
The comment for `InvalidDataHash` should say data script instead of validator script